api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -167,4 +178,13 @@ public void testClearsEraCommonsStatus() throws Exception {
     assertThat(user.getEraCommonsCompletionTime()).isNull();
   }
 
+  @Test
+  public void testSyncTwoFactorAuthStatus() throws Exception {
```

Comment:
i think the other case warrants a test case too. should be a quick add.

(setting 2fa completion time to false)